src/app/utilities/course/constants.ts

```diff
@@ -66,3 +75,15 @@ export const FILTER_KEYS: (keyof FilterCourseProps)[] = [
 ];
 
 export const COURSE_FETCH_REVALIDATE_PERIOD = 60 * 30; // In seconds
+
+export const COURSE_BENEFIT_SUPPORT_AVAILABLE = {
+  'Trained Staff': TrainedStaff,
+  'Universal Design for Learning': UniversalDesignforLearning,
+  'Inclusive policies': InclusivePolicies,
+  'Alternative assessment methods': AlternativeAssessmentMethods,
+  'Regular feedback mechanisms': Feedback,
+  'Sensory breaks': FiveSenses,
+  'Specialized academic advising': LearningTools,
+  'Accessible textbooks': ReadingMode,
+  'Inclusive teaching practices': TeachingPractice,
+};
```

Comment:
Labels are not ideal keys/identifier. Rather use keys like this - `TrainedStaff`, `DesignLearning` etc. which can map to an object containing relevant label, icon etc. or you can think of other ways utilizing keys.